src/theme/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { CssBaseline, createTheme, PaletteMode } from '@mui/material';
+import { ThemeProvider } from '@mui/material/styles';
+import { useMemo } from 'react';
+import { getDesignTokens } from './themeTokens';
+import { useAppSelector } from '../hooks/redux-hooks';
+import { getProfile } from '../stores/slices/authSlice';
+
+const ThemeWrapper = ({ children }: { children: React.ReactNode }) => {
+  const uiTheme = useAppSelector(getProfile)?.uiTheme || localStorage.getItem('ui_theme');
```

Comment:
It would be nice to have a utility function that extracts this specific value of the theme from local storage and returns a typed value. 

It is needed to avoid several things:
1) Pointing string as a key in many places (in case you want to rename the key of the theme you need to update all finds of that key)
2) Inline calling of `localStorage.getItem`. Having a separate named function will improve the readability of your code

Reply:
Good point!
Done !